### ISSUE TITLE
Build with protobuf in system 

### ISSUE THREAD

**Issue by acane77** [google/sentencepiece#787]
While using protobuf library in system (i.e., SPM_USE_BUILTIN_PROTOBUF=OFF, instead of `third_party/protobuf-lite`), hard-coded header file inclusion causes an error.

in `init.h:21`:

```cpp
#include "third_party/protobuf-lite/google/protobuf/message_lite.h"
```

it should be

```cpp
#include "google/protobuf/message_lite.h"
```

**Comment by taku910** [comment]
Will fix it in the next release

**Comment by taku910** [comment]
Fixed in v0.1.98
